pages/hydra/developer_resources/workflow_and_mediated_deposit/hyrax_1.0/workflow_cheat_sheet.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Workflow Cheat Sheet"
+keywords: Mediated Deposit, Workflow, Sipity
+categories: Workflow
+permalink: workflow_cheat_sheet.html
+folder: hydra/developer_resources/workflow_and_mediated_deposit/hyrax_1.0/workflow_cheat_sheet.md
+sidebar: home_sidebar
+tags: [development_resources]
+---
+
+# Handy commands
+
+### Get the active workflow for a given admin set
+```ruby
+a = AdminSet.first
+workflow = a.permission_template.available_workflows.where(active: true).first
+ => #<AdminSet id: "g732d8963", title: ["Laney Graduate School"], description: [], creator: ["redacted@fake.edu"], access_control_id: "ea6d2579-2f17-4e05-a348-f36716b783e5", representative_id: nil, thumbnail_id: nil>
+```
+
+### Get the workflow roles in a given workflow
+```ruby
+workflow_roles = Sipity::WorkflowRole.where(workflow_id: workflow.id)
+=> #<ActiveRecord::Relation [#<Sipity::WorkflowRole id: 10, workflow_id: 2, role_id: 1, created_at: "2017-06-05 13:52:09", updated_at: "2017-06-05 13:52:09">, #<Sipity::WorkflowRole id: 3, workflow_id: 2, role_id: 2, created_at: "2017-06-05 13:52:08", updated_at: "2017-06-05 13:52:08">, #<Sipity::WorkflowRole id: 1, workflow_id: 2, role_id: 3, created_at: "2017-06-05 13:52:08", updated_at: "2017-06-05 13:52:08">, #<Sipity::WorkflowRole id: 2, workflow_id: 2, role_id: 4, created_at: "2017-06-05 13:52:08", updated_at: "2017-06-05 13:52:08">]>
+workflow_roles.map { |role| Sipity::Role.where(id: role.role_id).first.name }
+ => ["managing", "depositing", "reviewing", "approving"]
+```
+
+### Print out all the users and their roles for a given workflow (useful for debugging)
+```ruby
+workflow = AdminSet.where(title: ["Laney Graduate School"]).first.permission_template.available_workflows.where(active: true).first
+User.all.each do |user|
+  puts "-----"
+  puts user.email
+  roles = Hyrax::Workflow::PermissionQuery.scope_processing_workflow_roles_for_user_and_workflow(user: user, workflow: workflow).pluck(:role_id)
```

Comment:
We may not want to encourage calling this method directly because it's part of the private API. I'm not :100: sure what its public equivalent is, but `scope_processing_workflow_roles_for_user_and_entity` should be close.